backend/dataall/modules/mlstudio/services/mlstudio_permissions.py

```diff
@@ -0,0 +1,44 @@
+"""
+Add module's permissions to the global permissions.
+Contains permissions for sagemaker ML Studio
+"""
+
+from dataall.db.permissions import (
+    ENVIRONMENT_ALL,
+    ENVIRONMENT_INVITED,
+    RESOURCES_ALL_WITH_DESC,
+    RESOURCES_ALL,
+    ENVIRONMENT_INVITATION_REQUEST,
+    TENANT_ALL,
+    TENANT_ALL_WITH_DESC
+)
+
+
+CREATE_SGMSTUDIO_USER = 'CREATE_SGMSTUDIO_USER'
+LIST_ENVIRONMENT_SGMSTUDIO_USERS = 'LIST_ENVIRONMENT_SGMSTUDIO_USERS'
```

Comment:
This one is used only in this file. I think it's save do delete it

Reply:
Added #TODO. I think we need to review all LIST_X_INENVIRONMENT permissions